src/neo/Utility.cs

```diff
@@ -29,6 +30,8 @@ public Logger()
             }
         }
 
+        public static event LogEventHandler Logging;
```

Comment:
`OnLog`?

Reply:
We usually don't add `on` before event name.